### PR TITLE
Week 1 booster session

### LINKED ISSUE #125: Week 1 booster session
AC
- [ ] Create walk-through doc for the outline below
- [ ] Hold session for new contributors to use and criticize walk-through
- [ ] Update doc based on learning

## outline
- git
  - install
  - setup
  - basic usage
- github
  - signup
  - basic walk-through
  - project policies

### PR DESCRIPTION
**Pull request by xipxape**
Closes #125

### CODE REVIEW

File: docs/learning/git.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+git switch main
+git pull origin main
+git switch -c me/my_branch-123
+# make change(s)
+git add .
+git commit -m"Make specific change"
+git push -u origin me/my_branch-123
+```
+
+Afterward, if additional commits are made, only `git push` is necessary since
```
Comment (codegold79) [inline docs/learning/git.md:67]: I would suggest referring to the lines that would be repeated for making the additional commits you mention. Something like,

```To commit more changes, repeat the three commands on lines 62-64.```

**Review by codegold79 — CHANGES_REQUESTED** [review]
This is a great start. I had some comments, but feel free to ignore/update per your discretion.

File: docs/learning/git.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+a copy of the repo is created under another organization/namespace), then
+modified, then have selected modifications "pulled" into the original
+repository for the improvements to be released publicly.
+
+### Installation
+
+- [Git - Getting Started](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+Side note: It is preferable to use Git version 2.28 or higher since it permits
+configuring `init.defaultBranch` which allows for using user-defined branches
```
[the remark this answers is not in the document]
Comment (daved) [reply docs/learning/git.md:22]: I'm just going to assume the user updates their machine/applications.

File: docs/learning/git.md (hunk 1 above)
Comment (daved) [reply docs/learning/git.md:67]: I've added a second snippet for clarity, and reduced the relevant note.

File: docs/learning/git.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+-b new-branch` should be `git switch -c new-branch`, and `git checkout
+existing-branch` should be `git switch existing-branch`. The usage of checkout
+to move to a specific commit (seen at 4:50) is correct. The git command will
+give notices to use `git switch` in the relevant cases and those messages can be
+seen in the video. Lastly, it is currently preferred to use the default branch
+name "main" rather than "master".
+
+### Interpretation
+
+A simple list of commands to start work on an issue.
```
[the remark this answers is not in the document]
Comment (daved) [reply docs/learning/git.md:56]: The user will hit the "no spaces" issue on their own, so I'd rather not mention it. The namespacing in the branch name has been dropped.

File: docs/learning/git.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+- [Paul Programming - What is Git?](https://youtu.be/OqmSzXDrJBk)
+
+The first video is "Git in principle", and the second is "Git in practice".
+
+The second video seems to imply that Git is centralized, which is in opposition
+to the description given in the first video. git is not dependent on a central
+repository, rather, releases are. It is common for projects to be "forked" (i.e.
+a copy of the repo is created under another organization/namespace), then
+modified, then have selected modifications "pulled" into the original
+repository for the improvements to be released publicly.
```
[the remark this answers is not in the document]
Comment (daved) [reply docs/learning/git.md]: Simplified (hopefully).

**Review by codegold79 — APPROVED** [review]
The changes you made read and look great. Thanks, @daved! LGTM

**Pull request merged**